### PR TITLE
Fix some phpdocs return types

### PR DESCRIPTION
**Pull request by hugofintecture**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Returns are string\|false not bool\|string
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     |no
| How to test?      | N/A
| Fixed ticket?     | N/A
| Related PRs       | N/A
| Sponsor company   | Fintecture

### CODE REVIEW

**Review by lartist — COMMENTED** [review]
Hi @hugofintecture, it's ok for me but you first need to fix phpstan error. Thank you.

**Review by FabienPapet — APPROVED** [review]

File: controllers/admin/AdminTranslationsController.php [hunk 1]
```diff
@@ -208,7 +208,7 @@ public function initMain()
         $arrayStreamContext = @stream_context_create(['http' => ['method' => 'GET', 'timeout' => 8]]);
 
         if ($langPacks = Tools::file_get_contents($this->link_lang_pack, false, $arrayStreamContext)) {
-            if ($langPacks != '' && $langPacks = json_decode($langPacks, true)) {
+            if ($langPacks = json_decode($langPacks, true)) {
```
Comment (matks) [inline controllers/admin/AdminTranslationsController.php:211]: If I am not wrong, if `$langPacks` is equal to empty string then the behavior is different? You changed the behavior?

Comment (hugofintecture) [reply controllers/admin/AdminTranslationsController.php:211]: The `if ($langPacks =` is already taking care of a possible empty value. You can check this example :) :
https://onlinephp.io/c/00674

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
@PrestaShop/qa-team there's nothing to test, updated comments and removed a not needed if, can we merge without QA?

**Comment by HanaRebaiQA** [comment]
Hello @kpodemski 

Ok for us. GO for testing it :) 

Thanks !

**Comment by lartist** [comment]
Thanks @hugofintecture

**Pull request merged**